### DISCUSSION TITLE
Support for Angular 19 LTS

### DISCUSSION THREAD

**Discussion by MGREMY** [nrwl/nx#29013] [Feature Requests]
I could be great to have support for the new angular version
![image](https://github.com/user-attachments/assets/98cff3c0-fa5f-485f-8498-ca5b175b8806)

**Comment by shadow1349** [comment]
I would love to upgrade the projects in my monorepo to Angular 19. Thanks!

**Comment by PieterjanDeClippel** [comment]
Seems that they're working on it here: https://github.com/nrwl/nx/pull/28847/commits

**Reply by shadow1349** [reply]
Awesome!

**Comment by anthonylhowardjr** [comment]
Can't wait for this!

**Comment by tejasv-tagline** [comment]
How can I create new workspace with Angular 19 project in NX?

**Comment by PawelOwczarekFalcon** [comment]
I'm looking further for ng19
